File: Python/PRAK502-2210817310009-Mario Franca Wijaya.py
def hitung(num1, num2):
    return (int(num1)- int(num2))
def mutlak(hasil):
    return abs(int(hasil))

a, b, c, d = input().split()
hasil = hitung(a, c) + hitung(b, d)
print("", mutlak(hasil))

File: C/PRAK502-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>
#include <math.h>

int hitung(int number1, int number2){
    return (number1 - number2);
}
int mutlak(int hasil){
    return abs(hasil);
}
int main(){
    int a,b,c,d, hasil;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);

    hasil = hitung(a,c) + hitung(b,d);
    printf("%d", mutlak(hasil));
}